### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by Salimer**
## What does this PR do?

This PR is raised to implement the following features 👍:

- [x]  Delete all the boilerplate from Create React App (text, images, styles).
- [x] Create a new directory called components.
- [x] Inside components, create a new Calculator.js file.
- [x] In Calculator.js, create a React component that matches the given disign.
- [x] Implement the design only (JSX and CSS).
- [x] Use functional components

## Description of the completed tasks 🙌

- [x] Added the HTML for the structure. 
- [x] Added the SCSS file for the styling.
- [x] Added the js file for DOM manipulation.
- [x] Used the linter tool that analyzes my source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. 💯

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <div className="button">2</div>
+        <div className="button">3</div>
+        <div className="button">0</div>
+        <div className="button">.</div>
+      </div>
+    </article>
+  );
+}
+
+export default Calculator;
```
Comment (TedLivist) [inline src/components/Calculator.js]: - [ ] Your calculator component should have at least one child component. Kindly create a new component in a new file and import it here.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import './Calculator.scss';
+
+function Calculator() {
+  return (
+    <article className="calculator-container">
+      <div className="screen">0</div>
+      <div className="operator">
+        <div className="button">/</div>
```
Comment (TedLivist) [inline src/components/Calculator.js]: - [x] This symbol should be &div;, not /, based on the design. If you do not have the sign on your keyboard consider using HTML entity code for it.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Salimer,

Good job so far!
✔️ Use of functional components
✔️ Nice design
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Kindly remove the `robot.txt` file from the `public` folder; the `logo.svg` file from the `src` folder as they are boilerplate files that won't be needed for this project and subsequent ones.

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Salimer** [comment]
Thank you very much @TedLivist for the thorough review. I addressed the required changes 🥳

**Review by Reem-lab — APPROVED** [review]
# Hi @Salimer ,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- [ ] Great job on the funtionality on the component💯
- [ ] Followed Gitflow 🔥
- [ ] No Linters error ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**